### PR TITLE
Chore/service status indications

### PR DESCRIPTION
**Pull request by joshenlim**
image

### CODE REVIEW

File: studio/components/interfaces/Home/ServiceStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+      isSuccess: isSuccessAuth,
+    },
+    {
+      name: 'Database',
+      isLoading: isLoadingPostgres,
+      isError: isErrorPostgres,
+      isSuccess: isSuccessPostgres,
+    },
+    {
+      name: 'Postgrest',
```
Comment (steve-chavez) [inline studio/components/interfaces/Home/ServiceStatus.tsx:103]: IMO it's clearer this way:
```suggestion
      name: 'PostgREST',
```

Comment (steve-chavez) [reply studio/components/interfaces/Home/ServiceStatus.tsx:103]: Alternatively just "REST" or "REST API"

File: studio/data/service-status/edge-functions-status-query.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+export type EdgeFunctionServiceStatusVariables = {
+  projectRef?: string
+}
+
+export async function getEdgeFunctionServiceStatus(
+  { projectRef }: EdgeFunctionServiceStatusVariables,
+  signal?: AbortSignal
+) {
+  const res = await get(`https://obuldanrptloktxcffvn.supabase.co/functions/v1/health-check`, {
```
Comment (alaister) [inline studio/data/service-status/edge-functions-status-query.ts:15]: Should this be a static URL? Or should `projectRef` be interpolated?

Comment (joshenlim) [reply studio/data/service-status/edge-functions-status-query.ts:15]: static URL! but good point - shouldn't need project ref here
imma keep it in the query key though, although we'd be pinging the same endpoint, but its to have RQ check once more each time the user switches projects

this check for edge functions is kinda like a v1 until we're able to make it more fine grained

**Review by alaister — APPROVED** [review]
LGTM!

I think it might also be good to include an action for the user to take when their services are unhealthy

**Pull request merged**